Static Framework Sample/static-framework-sample-app/Showcases/ErrorAnalysis/ErrorAnalysisViewController.swift

```diff
@@ -0,0 +1,64 @@
+
+import ContentsquareErrorAnalysisModule
+import UIKit
+
+/// This class showcases how to use Error Analysis to trace HTTP network errors
+class ErrorAnalysisViewController: UIViewController {
```

Comment:
I know it's done like this previously, but I was wondering if we can use the same file for all samples.
Let me check with the team if it's a good idea.

Reply:
Let's keep it as it is for now.

Reply:
Removed duplicates